compiler-core/src/exhaustiveness.rs

```diff
@@ -312,13 +313,13 @@ pub enum Pattern {
         name: EcoString,
     },
     Tuple {
-        elements: Vec<Id<Pattern>>,
+        elems: Vec<Id<Pattern>>,
```

Comment:
No abbreviations please, `elements` and `arguments`. Though perhaps `arguments` should be `values` or `fields`?

Reply:
Ok, fixed! I've renamed `args` to `fields`